src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -153,7 +153,14 @@ public function start()
         }
 
         // ok to try and start the session
-        if (!session_start()) {
+        $options = [];
+        // Subsequent calls to session_start() cause PHP to unconditionally
+        // [re-]set the cookie header, which may be undesirable.
+        // @see https://bugs.php.net/bug.php?id=38104#1491877141
+        if (true === $this->closed) {
+            $options['use_cookies'] = 0;
```

Comment:
I'm not sure if this is considered BC break or not... it's certainly unintuitive behaviour and I'd be surprised if anyone is depending on this broken behaviour, but still...